features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -537,6 +537,14 @@
             <!--AllowAdditionalParamsFromPostLogoutRedirectURI>false</AllowAdditionalParamsFromPostLogoutRedirectURI-->
         </OpenIDConnect>
 
+        <!--
+            FAPI Support.
+            Supported versions: IS 5.11.0 onwards
```

Comment:
As this was not part of 5.11.0 GA, it's incorrect to say the support is from 5.11.0 onwards

Reply:
noted.will remove